src/platform/tls_schannel.c

```diff
@@ -975,6 +975,8 @@ CxPlatTlsSecConfigCreate(
     case QUIC_CREDENTIAL_TYPE_CERTIFICATE_CONTEXT:
 #endif
     case QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE:
+    case QUIC_CREDENTIAL_TYPE_CERTIFICATE_FILE_PROTECTED:
+    case QUIC_CREDENTIAL_TYPE_CERTIFICATE_PKCS12:
```

Comment:
I'm a bit confused why these should be NOT_SUPPORTED. Is this supposed to kernel specific?

Reply:
Yes this chain is kernel specific. Which will never support these 2 methods, at least in the current state.

Reply:
Wait actually nevermind. I messed up here.